src/libs/TransactionUtils.js

```diff
@@ -389,6 +389,16 @@ function getValidWaypoints(waypoints, reArrangeIndexes = false) {
     return validWaypoints;
 }
 
+/**
+ * Returns the most recent transactions in an object
+ * @param {Object} transactions - Shaped with transctions as keys and inserted dated as values
```

Comment:
```suggestion
 * @param {Object} transactions - Shaped with transactions as keys and inserted dated as values
```

and should this be `transactionIDs`